google/cloud/forseti/scanner/scanners/config_validator_util/validator_client.py

```diff
@@ -75,12 +75,11 @@ def add_data(self, assets):
         except grpc.RpcError as e:
             # pylint: disable=no-member
             if e.code() == grpc.StatusCode.UNAVAILABLE:
-                raise errors.ConfigValidatorServerUnavailableError(
-                    e.message)
+                raise errors.ConfigValidatorServerUnavailableError(e)
```

Comment:
Earlier versions of Python2 had a message attribute on the base Exception class, however this was removed in later versions of Python2 and all of Python3. This was part of the exception message that was occurring when the Policy Library was not configured correctly.

```
AttributeError: '_Rendezvous' object has no attribute 'message'
```